app/models/agents/webhook_agent.rb

```diff
@@ -88,7 +88,7 @@ def receive_web_request(params, method, format)
         create_event(payload: payload)
       end
 
-      [response_message, code]
+      [interpolated(params)['response'] || interpolated['response'] || 'Event Created', code]
```

Comment:
To maintain compatibility with the current context... or not?

If you find it is not necessary, I remove.